doc/guides/build/kconfig/setting.rst

```diff
@@ -167,6 +167,22 @@ The application configuration can come from the sources below. By default,
 6. Otherwise, :file:`prj.conf` is used if it exists in the application
    directory
 
+Per default all configuration files will be taken from the application's
+configuration directory except for files with an absolute path that are
+given with the ``CONF_FILE`` argument.
```

Comment:
At line 156 we talk about the application directory.

But it should be application configuration directory, right?

Maybe even APPLICATION_CONFIGURATION_DIRECTORY to be clear that this is a precise term.

Reply:
We already have `APPLICATION_SOURCE_DIR` and `APPLICATION_BINARY_DIR` so I think sticking to `_DIR` fits nicely along that line. (which also includes CMAKE_CURRENT_SOURCE_DIR, CMAKE_CURRENT_LIST_DIR, etc.)

But with adjust to: `APPLICATION_CONFIGURATION_DIR` to be more precise.

And thanks for spotting the line 156, will fix.

Reply:
Sorry, used the wrong name.